src/MultiDimFit.cc

```diff
@@ -626,8 +637,12 @@ void MultiDimFit::doGrid(RooWorkspace *w, RooAbsReal &nll)
     }
 
     if (n == 1) {
+        if (verbose > 1) std::cout << "\nStarting n==1. The nll0 from initial fit: " << nll0 << std::endl;
         unsigned int points = pointsPerPoi.size() == 0 ? points_ : pointsPerPoi[0];
 
+        // Set seed for random points
+        srand(1);
```

Comment:
Not sure right now if this could have other knock-on effects, but perhaps it would be better to do this **only** if `pointsRandProf > 1` just to be safe

Reply:
Resolved.